src/App.css

```diff
@@ -0,0 +1,164 @@
+.twitter-app{
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  margin: 10px;
+}
+
+/* Side navbar*/
+.side-navbar{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 0;
+  flex-grow: 1;
+  max-width: 230px;
+}
+
+.logo{
+  width: 46px;
+  height: 2rem;
+  padding-bottom: 7px;
+}
+
+.navbar-list-item{
+  padding: 12px;
+}
+.navbar-list-item:hover{
+  color: rgba(29,161,242,1.00);
+}
+
+.navbar-list-item-logo{
+  display: inline-flex;
+}
+.navbar-list-item-logo svg{
+  height: 1.6rem;
+}
+
+.navbar-list-item-text{
+  display: inline-flex;
+  vertical-align: 5px;
+}
+.title{
+  font-size: 20px;
+  font-weight: 550; 
+  margin-left: 10px;
+}
+
+/* Buttons */
```

Comment:
I see here and in other places in App.css file that you added a comment and than the css rules.
I prefer to split components and also css/scss files by their role to avoid to long ones that will be hard to navigate for the long term.

For example: NewsFeed.scss is one file and keeping it like that with other components.